docs/sections/contributor_guide/fork_pr_model.rst

```diff
@@ -4,9 +4,9 @@ Fork and PR Model
 Overview
 --------
 
-Contributions to the ``workflow-tools`` project are made via a `Fork <https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/about-forks>`_ and `Pull Request (PR) <https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/about-pull-requests>`_ model. GitHub provides a thorough description of this contribution model in their `Contributing to projects` `Quickstart <https://docs.github.com/en/get-started/quickstart/contributing-to-projects>`_, but the steps, with respect to ``workflow-tools`` contributions, can be summarized as:
+Contributions to the ``workflow-tools`` project are made via a `Fork <https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/about-forks>`__ and `Pull Request (PR) <https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/about-pull-requests>`_ model. GitHub provides a thorough description of this contribution model in their `Contributing to projects` `Quickstart <https://docs.github.com/en/get-started/quickstart/contributing-to-projects>`_, but the steps, with respect to ``workflow-tools`` contributions, can be summarized as:
```

Comment:
In case you also run into it: Because there were two links in this `.rst` file here where the link text was just `Fork` (on this line, and again on line 9), `sphinx-build` complained about a redefinition. The solution was to use *two* underscores after the link to make it an "anonymous reference", which eliminated the warning.

Reply:
Interesting! I'm sure this'll come up again